src/utils/error.ts

```diff
@@ -51,7 +53,7 @@ export class CombinedError extends Error {
     response,
   }: {
     networkError?: Error;
-    graphQLErrors?: Array<string | GraphQLError | Error>;
+    graphQLErrors?: graphqlError[];
```

Comment:
This doesn't seem strictly necessary; If there's a rule for it, I'd say we should maybe disable it 🤔 It seems to be a good rule of thumb but often we really don't care about a type alias, especially when, like here, we only convert all of these and normalise them anyway

Reply:
Yep, this was just a linting flag, I'll switch it back 👍